### ISSUE TITLE
Smoke test timeouts

### ISSUE THREAD

**Issue by kylebgorman** [kylebgorman/wikipron#101]
For an example, see #100. I think these are all false positives...timeout should be increased substantially.

**Comment by jacksonllee** [comment]
My current workaround is to just make CircleCI run the builds again until everything's green. :-) (Just did that for your #100, but Latin is a tough beast...) Not the most urgent thing on our plate now -- I'll fiddle with the timeout setting, possibly with related performance issues, over the holidays.

**Comment by kylebgorman** [comment]
Yes, very low urgency.

**Comment by kylebgorman** [comment]
Starting to think we should disable these on CircleCI too. Not a single one of these has passed for me in the last week or two.

**Comment by jacksonllee** [comment]
I added the timeout in the smoke tests just in case the internet doesn't play nice and the smoke tests hang. But I forgot that we've turned on timeout in the scraping code, and so my concern isn't warranted anymore.

https://github.com/kylebgorman/wikipron/blob/5d7429cf9703f9cfa3cdd81370348654b5b10117/wikipron/scrape.py#L41

Resolving this ticket would mean just removing the single line of `@pytest.mark.timeout` decorator, nothing as complicated as getting into a rabbit hole about performance and such.
